nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/sqs/DeleteSQS.java

```diff
@@ -45,7 +46,8 @@
 public class DeleteSQS extends AbstractSQSProcessor {
 
     public static final PropertyDescriptor RECEIPT_HANDLE = new PropertyDescriptor.Builder()
-            .name("Receipt Handle")
+            .name("sqs-receipt-handle")
```

Comment:
@jvwing 

I think changing the name attribute at this point will break existing flows.